terraform/aks/workspace-variables/sandbox.tfvars.json

```diff
@@ -0,0 +1,44 @@
+{
+  "env_config": "sandbox",
+  "app_environment": "sandbox",
+  "key_vault_name": "s189p01-rtt-sbx-kv",
+  "key_vault_app_secret_name": "REGISTER-APP-SECRETS-SANDBOX",
+  "key_vault_infra_secret_name": "BAT-INFRA-SECRETS-PRODUCTION",
+  "cluster": "production",
+  "namespace": "bat-production",
+  "db_sslmode": "prefer",
+  "main_app": {
+    "main": {
+      "startup_command": ["/bin/sh", "-c", "STATEMENT_TIMEOUT=90s bundle exec rails db:migrate && unset STATEMENT_TIMEOUT && bundle exec rails server -b 0.0.0.0"],
+      "probe_path": "/ping",
+      "replicas": 2,
+      "memory_max": "1536Mi"
+    }
+  },
+  "worker_apps": {
+    "worker": {
+      "startup_command": ["/bin/sh", "-c", "bundle exec sidekiq -C config/sidekiq.yml"],
+      "replicas": 0,
+      "memory_max": "1Gi"
+    }
+  },
+  "azure_maintenance_window": {
+    "day_of_week": 0,
+    "start_hour": 2,
+    "start_minute": 0
+  },
+  "postgres_flexible_server_sku": "GP_Standard_D2ds_v4",
+  "azure_enable_backup_storage": false,
+  "deploy_temp_data_storage_account": true,
+  "azure_tempdata_storage_account_name": "s189p01registerpdttmp",
+  "azure_resource_group_name": "s189p01-rtt-pdt-rg",
+  "azure_storage_account_replication_type": "RAGRS",
+  "gov_uk_host_names": [
+    "sandbox.register-trainee-teachers.service.gov.uk",
+    "sandbox.register-trainee-teachers.education.gov.uk"
+  ],
+  "enable_alerting": false,
+  "enable_monitoring": true,
```

Comment:
do we want monitoring for this env? We don't monitor productiondata.

Reply:
Probably not right now, have disabled